scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeOps.scala

```diff
@@ -609,4 +609,11 @@ object TreeOps {
       }
     }
 
+  def isChildOfCaseClause(tree: Tree): Boolean =
+    findTreeWithParent(tree) {
+      case t: Case => Some(tree ne t.body)
```

Comment:
What is reason to not use just `==`?

Reply:
i don't really know how `==` is implemented for trees but i'd like to check that it's *the same exact* instance here rather than something which looks the same.